thunder.profile

```diff
@@ -334,6 +334,31 @@ function thunder_modules_installed($modules) {
   }
 }
 
+/**
+ * Implements hook_modules_uninstalled().
+ */
+function thunder_modules_uninstalled($modules) {
+  // Import the content view if it was deleted during module uninstalling.
+  // This could happen if content_lock was uninstalled and the content view
+  // contained content_lock fields at that time.
+  if (in_array('content_lock', $modules)) {
```

Comment:
Always best to use the third argument of in_array()... just set it to TRUE for type safe checking - otherwise some truly odd things are possible. See, for example, https://www.drupal.org/forum/support/module-development-and-code-questions/2009-09-10/epic-wtf-with-in_array-please-help